kratos/processes/vtk_output_process.cpp

```diff
@@ -354,7 +354,7 @@ void VtkOutputProcess::WriteElementData(ModelPart &model_part)
         outputFile << "SCALARS SPLIT_ELEMENT float 1\nLOOKUP_TABLE default\n";
 
         // write element results for active
-        for (ModelPart::ElementIterator elem_i = model_part.ElementsBegin(); elem_i != model_part.ElementsEnd(); ++elem_i)
+        for (ModelPart::ElementIterator elem_i = rModelPart.ElementsBegin(); elem_i != rModelPart.ElementsEnd(); ++elem_i)
```

Comment:
Is this code going to be used eventually?

Reply:
No I will remove this !